docs/glossary.md

```diff
@@ -0,0 +1,168 @@
+ Glossary of Technical Terms
+---
+
+## A
+- [Account Aliases](#account-aliases)
+- [Admin Transaction](#admin-transaction)
+- [Auto Associations](#asynchronous)
+- [Auto-Renew Accounts](#auto-renew-accounts)
+- [Address Book](#address-book)
+- [Application Accounts](#application-accounts)
+
+## B
+- [BLS](#bls)
+- [Block Proof](#block-proof)
+- [Block Signature](#block-signature)
+- [Block Stream](#block-stream)
+
+## C
+- [Civilian Accounts](#civilian-accounts)
+- [Config.txt](#config.txt)
+- [Congestion Pricing](#congestion-pricing)
+- [Consensus Time](#consensus-time)
+
+## D
+- [Delegate Call](#delegate-call)
+
+## E
+- [EOA](#eoa)
+
+## H
+- [Hollow Accounts](#hollow-accounts)
+
+## L
+- [Leaky Tests](#leaky-tests)
+
+## P
+- [Precompiles](#precompiles)
+
+## S
+- [Security Model V2](#security-model-v2)
+- [Sidecars](#sidecars)
+- [Synthetic Transaction](#synthetic-transactions)
+- [System Contracts](#system-contracts)
+- [System Transaction](#system-transaction)
+
+## T
+- [Token Allowance](#token-allowance)
+
+## W
+- [Weibar](#weibar)
+
+---
+
+## Address Book
+**Address Book**: The _**network**_ address book is data structure that contains information (such as node ID, alias, IP address, and weight) for all nodes in the network, 
+not just consensus, but also block nodes, future validator nodes and more.
+The address book is stored as a file on disk at the moment but will be stored purely in state as part of Dynamic Address Book phase 3.	Disambiguation: There is also an "address book" in files 0.0.101 and 0.0.102 which is different to the "network address book" which are manually managed by devops.
+
+
+## Account Aliases
+**Account Aliases**: These are identifiers which are public key values, and are mostly 33 byte EVM address values. The "triplet" form (e.g. 0.0.18273892) is the simpler and easier to "type".
```

Comment:
The aliases can be protobuf serialized keys of ED25519 or ECDSA Keys or EVM address

Reply:
That's correct, there are more options than just a public key.

An EVM address is a 20 byte keccak-256 of the ECDSA account key.
The "synthetic" EVM address (built from the account ID) is a bit weird as it's not stored as an alias, but can be used in the alias field of the `AccountID` message.
That bit of semantic gymnastics is courtesy of wanting EVM addresses but not wanting to implement the multiple-EVM-address features in HIP-631.

Probably way too much for a "glossary" (and not yet final or published), but there is good detail [here](https://github.com/hashgraph/hedera-protobufs/blob/pbj-storage-spec-review/documents/api/services/basic_types.md#proto-AccountID)